browserscripts/browser/appConstants.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+  requires: { privilege: true },
+  function: function() {
+    const { AppConstants } = ChromeUtils.import(
+      'resource://gre/modules/AppConstants.jsm'
+    );
+    return {
```

Comment:
Why are we stringifying?  Can't we pass back some (potentially sanitized) object?